src/lib/Behat/PageObject/TrashPage.php

```diff
@@ -48,13 +54,16 @@ public function verifyElements(): void
         $this->adminList->verifyVisibility();
     }
 
-    public function verifyIfItemInTrash(string $itemType, string $itemName): void
+    public function verifyIfItemInTrash(string $itemType, string $itemName, bool $elementShouldExist): void
     {
+        $isElementInTrash = $this->isTrashEmpty() ||
```

Comment:
This doesn't look right.

$isElementInTrash = $this->isTrashEmpty() -> if isTrashEmpty = true, then $isElementInTrash = true? For empty Trash the `isElementInTrash` should be false.

I'd start with the assertion:
`Assert::assertEquals($elementShouldExist, $isElementInTrash, ...)`
and work on the condition so that it makes sense.

I suppose that there's a negation missing somewhere, which makes the whole thing work (because the assertion is also inverted incorrectly).